experiments/xls/date_parsers.py

```diff
@@ -4,7 +4,7 @@
 
 from pandas._libs.tslibs.timestamps import Timestamp
 
-ValidDateType = Union[str, Timestamp]
+ValidDateType = Union[str, Timestamp, datetime.datetime, datetime.date]
 
 
 class AbstractDateParser:
```

Comment:
raise_value_error message should include DD-MM-YYYY format.
"Date must be in format DD.MM.YYYY, DD/MM/YYYY or DD-MM-YYYY [...]"